### ISSUE TITLE
Scan with Accessibility Insights for Web reports issues with igx-grid

### ISSUE THREAD

**Issue by kacheshmarova** [IgniteUI/igniteui-angular#7037]
## Description  
FastPass with  "Accessibility Insights for Web" tool  reports issues related to igx-grid.

![AIWReport](https://user-images.githubusercontent.com/13034836/78155027-a0596500-7445-11ea-9e60-46dcaf509fa3.PNG)

This sample has Paging feature enabled. However, I believe that issues will be present for other features as well.
 * igniteui-angular version: 9.0
 * browser: Chrome

## Steps to reproduce  

1. Enable [Accessibility Insights for Web](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni?hl=en) in Chrome
2. Run the attached sample in Chrome.
3. Click this button 
![Capturhhhe](https://user-images.githubusercontent.com/13034836/78155818-9c7a1280-7446-11ea-899e-e272d05dbb95.PNG)
 and select "FastPass" in order to run a scan.
4. Observe the results
## Result  
![Capture](https://user-images.githubusercontent.com/13034836/78156124-fe3a7c80-7446-11ea-9e14-c54f0cb55dad.PNG)
If expanded every section gives additional information - Path, Snippet and How to fix. For example:
![Capppture](https://user-images.githubusercontent.com/13034836/78158400-dac50100-7449-11ea-8e4f-afc53ce145c6.PNG)

## Expected result  
The "FastPass" to run without failed instances.

## Attachments  

[grid-paging.zip](https://github.com/IgniteUI/igniteui-angular/files/4416246/grid-paging.zip)

**Comment by zdrawku** [comment]
Thank you @kacheshmarova! I've made a PR with the changes, although keep in mind that another issue should be logged for the contrast aria problems for the `header text colors`.

For the time being my suggestion is to use a custom grid theme (app.component.scss from the provided sample):

```scss
@import '~igniteui-angular/lib/core/styles/themes/index';

@include igx-core();

$my-grid-theme: igx-grid-theme(
    $header-text-color: #000000
);

:host {
  ::ng-deep {
    @include igx-grid($my-grid-theme);
  }
}
```

Result (custom theme + PR)

![image](https://user-images.githubusercontent.com/11193764/79476436-a63c7200-8011-11ea-915b-062374aaf8e6.png)
